.github/workflows/build-magma-windows.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         with:
           path: magma_*_cuda*_*.7z
           overwrite: true
+          name: artifact_${{ matrix.cuda_version }}_${{ matrix.config }}
```

Comment:
With this, the artifact is not uploaded to a zip with a different name:

```
2024-11-07T15:36:52.7793012Z Artifact artifact_126_Release.zip successfully finalized. Artifact ID 2158495441
2024-11-07T15:36:52.7794688Z Artifact artifact_126_Release has been successfully uploaded! Final size is 62089698 bytes. Artifact ID is 2158495441
2024-11-07T15:36:52.7804031Z Artifact download URL: https://github.com/pytorch/builder/actions/runs/11724637191/artifacts/2158495441
```

That's fine, but how does that help?
The push job downloads the artifact and expects something called `magma_*_cuda*.7z` - would that still work?

Reply:
Yes this is upload to GHA. So that artifact can be downloaded from the workflow page